### PR TITLE
feat: add AIRAC command

### PR DESCRIPTION
**Pull request by peregrine-developments**
## Description

Explains the limitation of free SimBrief accounts to AIRAC 2003 and potential issues it can cause.

## Test Results

image
(updated)

## Discord Username

pererry#8489

### CODE REVIEW

**Comment by tracernz** [comment]
Usually you will get either NOT IN DATABASE or AWY/WPT MISMATCH during the import.

**Comment by benw202** [comment]
It may also be worth specifying that you can't then use the SimBreif import feature, although obvious!

**Comment by peregrine-developments** [comment]
Updated command output!
image

**Comment by benw202** [comment]
> Updated command output! image

Apologies, I didn't specify properly, you will be able to use the import feature, but you will get the errors. If you use one of the other route generators, you then can't import your flight plan (have to fill it in manually).

**Comment by peregrine-developments** [comment]
Good stuff! I added clarification the manual route input as well

File: src/commands/support/airac.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export const airac: CommandDefinition = {
+    name: 'airac',
+    description: 'Provides information about free SimBrief account AIRAC limitations',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const airacEmbed = makeEmbed({
+            title: 'FlyByWire Support | SimBrief AIRACs',
+            description: makeLines ([
+                'Free SimBrief accounts are limited to generating routes valid for AIRAC 2003, while MSFS currently uses AIRAC 2203. '
```
Comment (Valastiri) [inline src/commands/support/airac.ts:13]:
```suggestion
                'Free SimBrief accounts are limited to generating routes valid for AIRAC 2003, while MSFS regularly updates to the latest AIRAC available. '
```

Makes this command more future proof against AIRAC changes.

File: src/commands/support/airac.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                + 'This can lead to route incompatibilies and various error messages on the MCDU, including "NOT ALLOWED", "NOT IN DATABASE", and "AWY/WPT MISMATCH". ',
+                '',
+                'Any of these errors mean your route is not valid for the current AIRAC, and cannot be properly used with SimBrief import!',
```
Comment (Valastiri) [inline src/commands/support/airac.ts:16]:
```suggestion
                'Any of these errors mean your route is not valid for the current AIRAC, and cannot be properly used alongside our SimBrief import feature!',
```

File: src/commands/support/airac.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                '',
+                'Some alternative route generators are available, that can be used with manual route input:',
+                '',
+                '[RouteFinder](http://rfinder.asalink.net/free/)',
+                '[Flight Plan Database](https://flightplandatabase.com/)',
+                '[Online Flight Planner](http://onlineflightplanner.org/)',
```
Comment (Valastiri) [inline src/commands/support/airac.ts:22]: See Discord for my comments

**Review by Valastiri — CHANGES_REQUESTED** [review]
Somethings to consider and copy changes.

File: src/commands/support/airac.ts (hunk 1 above)
Comment (peregrine-developments) [reply src/commands/support/airac.ts:13]: This is an especially good suggestion for futureproofing

File: src/commands/support/airac.ts (hunk 3 above)
Comment (Valastiri) [reply src/commands/support/airac.ts:22]: Disregard this

**Review by Valastiri — APPROVED** [review]

**Review by NUTTTTTTTT — CHANGES_REQUESTED** [review]
image
While the other two errors are valid points, NOT ALLOWED comes up in a number of circumstances, and may not always mean the route is invalid. If NOT ALLOWED does definitely come up with simbrief import errors (never seen it personally), I'd recommend changing the wording to be a little less definitive when saying the route is invalid.

**Comment by peregrine-developments** [comment]
confirmed: NOT ALLOWED error can show during route import in cases such as EDDB, which refers to a different airport in 2203 than in 2003

**Review by NUTTTTTTTT — APPROVED** [review]

**Pull request merged**